src/common/KeyboardAvoider.js

```diff
@@ -9,6 +9,25 @@ type Props = $ReadOnly<{|
   children: React$Node,
   style?: ViewStyleProp,
   contentContainerStyle?: ViewStyleProp,
+
+  /** How much the top of `KeyboardAvoidingView`'s layout *parent*
```

Comment:
Should say `KeyboardAvoider` here, I think.

(They happen to work out to the same thing, but that's only evident from looking at the implementation of `KeyboardAvoider`.)